src/components/data_submission/ds_study_information.js

```diff
@@ -87,26 +87,29 @@ export default function DataSubmissionStudyInformation(props) {
     h(FormField, {
       isRendered: !isEmpty(user),
       id: 'dataSubmitterName',
-      title: 'Data Submitter Name',
+      title: 'Data Submitter Name ',
+      helpText: ' The individual completing this form will be saved with the study.',
```

Comment:
Instead of using spaces it might be worth using something like 2em padding instead.